src/custom/components/swap/EthWethWrap/helpers.ts

```diff
@@ -35,11 +38,29 @@ export const _getAvailableTransactions = ({
 }: {
   nativeBalance?: CurrencyAmount<Currency>
   nativeInput?: CurrencyAmount<Currency>
-  singleTxCost: CurrencyAmount<Currency>
+  singleTxCost?: CurrencyAmount<Currency>
 }) => {
-  if (!nativeBalance || !nativeInput || nativeBalance.lessThan(nativeInput.add(singleTxCost))) return null
+  if (!nativeBalance || !nativeInput || !singleTxCost || nativeBalance.lessThan(nativeInput.add(singleTxCost))) {
+    return null
+  }
 
   // USER_BALANCE - (USER_WRAP_AMT + 1_TX_CST) / 1_TX_COST = AVAILABLE_TXS
   const txsAvailable = nativeBalance.subtract(nativeInput.add(singleTxCost)).divide(singleTxCost)
   return txsAvailable.lessThan('1') ? null : txsAvailable.toSignificant(1)
 }
+
+export function _estimateTxCost(gasPrice: ReturnType<typeof useGasPrices>, native: Currency | undefined) {
```

Comment:
i think is a bit strange to expose a function starting with an underscore. I assume this was a "private function" convention. If it's useful to expose, maybe should be without?

Reply:
Also, not sure if this should live in a more generic place instead of inside swap -> wrapEth